### PR TITLE
Available Monitoring oschecks

### PR DESCRIPTION
**Pull request by jbadiapa**
Explained the usege of the oschecks
Added a Sensu example with a detail description of the params (Client,check and security)

### CODE REVIEW

File: availability-monitoring-oschecks.txt [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+. Ensure the volume was deleted
+
+[source,bash]
+oschecks-check_cinder_volume  --username USER --password PASSWORD --auth_url AUTH_URL --tenant TENANT
+
+//.oschecks-check_ceph_health        
+  
+//[source,bash]
+//oschecks-check_ceph_health --os-username USER --os-password PASSWORD --os-tenant-name TENANT  --os-auth-url AUTH_URL
+
```
Comment (paramite) [inline availability-monitoring-oschecks.txt]: I don't think we need OpenStack credentials when testing Ceph. Are you sure about this?

File: availability-monitoring-oschecks.txt [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+. Delete the volume
+. Ensure the volume was deleted
+
+[source,bash]
+oschecks-check_cinder_volume  --username USER --password PASSWORD --auth_url AUTH_URL --tenant TENANT
+
+//.oschecks-check_ceph_health        
+  
+//[source,bash]
+//oschecks-check_ceph_health --os-username USER --os-password PASSWORD --os-tenant-name TENANT  --os-auth-url AUTH_URL
```
Comment (paramite) [inline availability-monitoring-oschecks.txt]: I don't think we need OpenStack credentials when testing Ceph. Are you sure about this?

File: availability-monitoring-oschecks.txt [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+
+
+//.oschecks-check_ceph_df            
+  
+//[source,bash]
+//oschecks-check_ceph_df --os-username USER --os-password PASSWORD --os-tenant-name TENANT  --os-auth-url AUTH_URL
+
```
Comment (paramite) [inline availability-monitoring-oschecks.txt]: Ditto.

File: availability-monitoring-oschecks.txt [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+//.oschecks-check_ceph_health        
+  
+//[source,bash]
+//oschecks-check_ceph_health --os-username USER --os-password PASSWORD --os-tenant-name TENANT  --os-auth-url AUTH_URL
+
+
+//.oschecks-check_ceph_df            
+  
+//[source,bash]
+//oschecks-check_ceph_df --os-username USER --os-password PASSWORD --os-tenant-name TENANT  --os-auth-url AUTH_URL
```
Comment (paramite) [inline availability-monitoring-oschecks.txt]: Ditto.

File: availability-monitoring-oschecks.txt (hunk 2 above)
Comment (jbadiapa) [inline availability-monitoring-oschecks.txt]: I haven't tested so far that's why it's still commented

Comment (paramite) [inline availability-monitoring-oschecks.txt]: I would suggest to remove all checks you did not verified. Proper documentation later is better than false one now.

**Comment by jbadiapa** [comment]
OK, 
I agree, It's better not saying anything
Added ceph oschecks.

File: availability-monitoring-oschecks.txt [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+. Create a volume
+. Check the status of the volume
+. Delete the volume
+. Ensure the volume was deleted
+
+[source,bash]
+oschecks-check_cinder_volume  --username USER --password PASSWORD --auth_url AUTH_URL --tenant TENANT
+
+.oschecks-check_ceph_health        
+It only execute the command *ceph healht* and parse the result to get a sensu result (0,1,2,3)
```
Comment (paramite) [inline availability-monitoring-oschecks.txt]: s/healht/health/

**Comment by paramite** [comment]
One small nit, but otherwise +1

**Pull request merged**